packages/block-library/src/group/edit.native.js

```diff
@@ -104,9 +120,17 @@ export default compose( [
 			isLastInnerBlockSelected = totalInnerBlocks === blockIndex;
 		}
 
+		const selectedBlockClientId = getSelectedBlockClientId();
+		const isSelected = selectedBlockClientId === clientId;
+
+		const parentId = getBlockRootClientId( clientId );
+		const parentBlockAlignment = getBlockAttributes( parentId )?.align;
+
 		return {
 			hasInnerBlocks,
 			isLastInnerBlockSelected,
+			isSelected,
```

Comment:
As far as I can see the `isSelected` should be already in props

Reply:
Right, nice catch